recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
-{% set version = "0.12.11" %}
-{% set sha256 = "a5b9fcc986b184db101aa280b42ecdcdfc524892596f606858e0b7a8b4d9e144" %}
+{% set version = "0.14.0" %}
+{% set sha256 = "7db1195b41c81f8274a7bbd97c956f44e8348265a1bc7641c37dfebc39f0c938" %}
 
 package:
   name: cachecontrol-split
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/C/CacheControl/CacheControl-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/C/CacheControl/cachecontrol-{{ version }}.tar.gz
```

Comment:
Was it failing because of the uppercase name? And why is everything else still capitalized (asking for what the consistency should be)?

Reply:
It was failing due to the capital letters. I double checked conda-forge and used what they had. I'm not super inclined to change anymore at the risk of breaking the URL.

Reply:
The package name is CacheControl because that's the original name on PyPI, see https://pypi.org/project/CacheControl/. The files are named differently and we can't do much about it. To see the list of files available, see https://pypi.org/project/CacheControl/#files. The Python ecosystem is case insensitive when it comes to package names, so it doesn't really matter. What's important here is to keep the package name like it is and use the filenames that are on PyPI (since we can't change them because we don't own them).